example/service/sqs/mockingClientsForTests/errorHandlingExample.go

```diff
@@ -0,0 +1,126 @@
+//go:build example
```

Comment:
if this example is sqs-specific, would it make sense to name the file that way? e.g. `sqsExample.go` or `sqsErrorHandlingExample.go`?